hardware_frame_context.go

```diff
@@ -0,0 +1,48 @@
+package astiav
+
+//#include <libavcodec/avcodec.h>
```

Comment:
This is weird to me that you don't need 
```go
//#cgo pkg-config: libavutil libavcodec
//#include <libavcodec/avcodec.h>
//#include <libavutil/hwcontext.h>
```
here 🤔

Reply:
I believe `#cgo` directives (including `#cgo CPPFLAGS`, `#cgo LDFLAGS`, etc) only need to specified once per package, whereas `#include` needs to be specified every file where it's needed.

Not sure about what is the best practice tho, but my preference both for my projects and projects at work is to add a `cgo.go` file to include all relevant cgo flags for a more centralized and easier management.

Reply:
Understood. Can you still add the 3 lines? I'll take some time afterward to clean things up 👍

Reply:
In fact can you only add the 2 `include` directives?